src/features/components/atoms/tab/types.ts

```diff
@@ -4,7 +4,7 @@ import { TString } from 'types/generics'
 
 export type TabItemTypes = {
     title: TString
-    component?: React.ReactElement
+    component?: React.ReactElement | React.ReactElement[]
```

Comment:
`component?: React.ReactElement[]`

As this props is only used for keys under Localize, React.ReactElement[] should be fine.